openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -169,19 +169,20 @@ class Config:
     or `independent` (independently sampled lambda windows).
     Default `repex`.
     """
-    online_analysis_interval: Optional[int] = None
+    online_analysis_interval: Optional[int] = 200
     """
     Interval at which to perform an analysis of the free energies.
     At each interval the free energy is estimate and the simulation is
     considered complete if the free energy estimate is below
     ``online_analysis_target_error``. If set, will write a yaml file with
     real time analysis data. Default `None`.
     """
-    online_analysis_target_error = 0.1 * unit.boltzmann_constant * unit.kelvin
+    online_analysis_target_error = 0.0 * unit.boltzmann_constant * unit.kelvin
```

Comment:
are these two settings that every 200 steps you check if the target error is below 0.0?  There's a `unit.timestep` for denoting steps if that's the case, else it's `unit.picosecond` maybe?

Reply:
Or more generally `Interval at which` is ambiguous and could mean either simulated time or timesteps so let's rephrase